tests/dynamics/test_bh_near_smbh.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+"""Test for bh_near_smbh function
+    in physics/dynamics.py
+"""
+######## Imports ########
+#### Standard Library ####
+import time
+#### Third-Party ####
+import numpy as np
+#### Homemade ####
+#### Local ####
+from mcfacts.physics.dynamics import bh_near_smbh
+
+######## Setup ########
+INPUT_DATA = {
```

Comment:
Please utilize the `INPUT_PARAMETERS` dictionary in `McFACTS/conftest.py` for these values.